device/src/state_machines/no_session.rs

```diff
@@ -323,9 +314,9 @@ where
             Event::TimeoutFired => {
                 let rx_config = radio::RfConfig {
                     frequency: self.shared.region.get_join_accept_frequency1(),
-                    bandwidth: self.shared.region.get_bandwidth(),
-                    spreading_factor: self.shared.region.get_spreading_factor(),
-                    coding_rate: self.shared.region.get_coding_rate(),
+                    bandwidth: radio::Bandwidth::_500KHZ,
```

Comment:
These should use the settings from the region I think?

Reply:
> Looks good to me at least, there are a few places where it should use settings from config instead of hardcoded values (I _think_).

Thanks looks like I missed a few of these!